src/main/java/seedu/address/logic/commands/DeleteByIndexCommand.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.book.Book;
+
+/**
+ * Deletes a book identified using it's displayed index from the catalog.
+ */
+public class DeleteByIndexCommand extends DeleteCommand {
+
+
```

Comment:
Do we need a message usage for each command?

Reply:
Message usage is in the parent class `DeleteCommand`